packages/@aws-cdk/aws-kinesisfirehose/test/delivery-stream.test.ts

```diff
@@ -178,6 +179,100 @@ describe('delivery stream', () => {
         },
       });
     });
+    test('metricIncomingBytes', () => {
+      const deliveryStream = new firehose.DeliveryStream(stack, 'Delivery Stream', {
+        destinations: [mockS3Destination],
+      });
+
+      const metric = deliveryStream.metricIncomingBytes();
+
+      expect(metric).toMatchObject({
+        account: stack.account,
+        region: stack.region,
+        namespace: 'AWS/Firehose',
+        metricName: 'IncomingBytes',
+        statistic: cloudwatch.Statistic.AVERAGE,
+        dimensions: {
+          DeliveryStreamName: deliveryStream.deliveryStreamName,
+        },
+      });
+    });
+
+    test('metricIncomingRecords', () => {
+      const deliveryStream = new firehose.DeliveryStream(stack, 'Delivery Stream', {
+        destinations: [mockS3Destination],
+      });
+
+      const metric = deliveryStream.metricIncomingRecords();
+
+      expect(metric).toMatchObject({
+        account: stack.account,
+        region: stack.region,
+        namespace: 'AWS/Firehose',
+        metricName: 'IncomingRecords',
+        statistic: cloudwatch.Statistic.AVERAGE,
+        dimensions: {
+          DeliveryStreamName: deliveryStream.deliveryStreamName,
+        },
+      });
+    });
+
+    test('metricBackupToS3Bytes', () => {
+      const deliveryStream = new firehose.DeliveryStream(stack, 'Delivery Stream', {
+        destinations: [mockS3Destination],
+      });
+
+      const metric = deliveryStream.metricBackupToS3Bytes();
+
+      expect(metric).toMatchObject({
+        account: stack.account,
+        region: stack.region,
+        namespace: 'AWS/Firehose',
+        metricName: 'BackupToS3.Bytes',
+        statistic: cloudwatch.Statistic.AVERAGE,
+        dimensions: {
+          DeliveryStreamName: deliveryStream.deliveryStreamName,
+        },
+      });
+    });
+
+    test('metricBackupToS3DataFreshness', () => {
+      const deliveryStream = new firehose.DeliveryStream(stack, 'Delivery Stream', {
+        destinations: [mockS3Destination],
+      });
+
+      const metric = deliveryStream.metricBackupToS3DataFreshness();
+
+      expect(metric).toMatchObject({
+        account: stack.account,
+        region: stack.region,
+        namespace: 'AWS/Firehose',
+        metricName: 'BackupToS3.DataFreshness',
+        statistic: cloudwatch.Statistic.AVERAGE,
+        dimensions: {
+          DeliveryStreamName: deliveryStream.deliveryStreamName,
+        },
+      });
+    });
+
+    test('metricBackupToS3Records', () => {
+      const deliveryStream = new firehose.DeliveryStream(stack, 'Delivery Stream', {
+        destinations: [mockS3Destination],
+      });
```

Comment:
These 3 lines are repeated in every single test.

How about adding a `describe()` here, and then a `beforeEach()` in it to get rid of this duplication?